### ISSUE TITLE
Problem in training 

### ISSUE THREAD

**Issue by scpaliuhc** [RanTaimu/Adversarial-Transformation-Network#2]
I just increased the value of training_epochs in atn.py and executed the command "python atn.py --train" with a new path for saving the model. But I found that the loss value of each epoch didn't change a lot:
Eopch 1 completed. loss = 99.92884826660156
Eopch 2 completed. loss = 98.98079681396484
Eopch 3 completed. loss = 99.99166870117188
Eopch 4 completed. loss = 99.44509887695312
Eopch 5 completed. loss = 98.99491119384766
Eopch 6 completed. loss = 100.26902770996094
Eopch 7 completed. loss = 99.92530822753906
Eopch 8 completed. loss = 99.55029296875
Eopch 9 completed. loss = 99.61663818359375
Eopch 10 completed. loss = 100.05928039550781
Eopch 11 completed. loss = 99.95893859863281
Eopch 12 completed. loss = 97.95149230957031
Eopch 13 completed. loss = 99.51895904541016
Eopch 14 completed. loss = 99.58808898925781
Eopch 15 completed. loss = 101.28242492675781
Eopch 16 completed. loss = 100.20669555664062
Eopch 17 completed. loss = 100.06300354003906
Eopch 18 completed. loss = 101.12290954589844
Eopch 19 completed. loss = 100.303955078125
Eopch 20 completed. loss = 100.51847839355469
Eopch 21 completed. loss = 100.51861572265625
Eopch 22 completed. loss = 100.18065643310547
Eopch 23 completed. loss = 98.83200073242188
Eopch 24 completed. loss = 101.06179809570312
Eopch 25 completed. loss = 99.15919494628906
Eopch 26 completed. loss = 100.65020751953125
Eopch 27 completed. loss = 99.72256469726562
Eopch 28 completed. loss = 99.38571166992188
Eopch 29 completed. loss = 100.00604248046875
Eopch 30 completed. loss = 99.70945739746094

After 30 epochs, I executed "python atn.py" for test, but there is nothing in the images generated by model._autoencoder. All these generated images are pure black.

I only changed the value of epochs and the path that the model will be saved in.  How did you train the model you provided in this project？

**Comment by kunal266** [comment]
same here, can someone help
